tests/e2escenarios/e2e_images_test.go

```diff
@@ -85,15 +86,11 @@ var _ = Describe("odo supported images e2e tests", func() {
 		Expect(cmpLst).To(ContainSubstring("Not Pushed"))
 	}
 
-	Context("odo supported images deployment", func() {
-		It("Should be able to verify the openjdk18-openshift image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.Join("redhat-openjdk-18", "openjdk18-openshift:latest"), "java:8", project)
-			verifySupportedImage(filepath.Join("redhat-openjdk-18", "openjdk18-openshift:latest"), "openjdk", "java:8", project, appName, context)
-		})
-
-		It("Should be able to verify the openjdk-11-rhel7 image", func() {
-			oc.ImportImageFromRegistry("registry.access.redhat.com", filepath.Join("openjdk", "openjdk-11-rhel7:latest"), "java:8", project)
-			verifySupportedImage(filepath.Join("openjdk", "openjdk-11-rhel7:latest"), "openjdk", "java:8", project, appName, context)
+	Context("odo supported images deployment on x86", func() {
```

Comment:
what about other platform? you are saying it's x86 ...

Reply:
@jichenjc Those images are only available for x86. I don't replace it with ubi/nodejs but pick these up to the specific context. I think it a good solution to not mix x86 and s390x together.

Reply:
I am only talk about `power` :)

Reply:
> I am only talk about `power` :)
I got it. I will update the condition to `amd64` .